plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/backstageRequest.ts

```diff
@@ -26,6 +26,7 @@ export function createHttpBackstageAction(options: { config: Config }) {
     method: Methods;
     headers?: Headers;
     params?: Params;
+    body?: string;
```

Comment:
Should this be `string | object`?

Reply:
It cannot be an object. It can be `string`, `string` with JSON or `undefined`

Reply:
But I did a bit refactoring and created a type alias for make it more readable.